From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tri Vo <trong@google.com>
Date: Tue, 16 Jul 2019 16:27:17 -0700
Subject: ANDROID: gki_defconfig: enable more configs

Enable following configs:
CONFIG_RAS
CONFIG_EDAC
CONFIG_SPI
CONFIG_DMADEVICES
CONFIG_POWER_AVS
CONFIG_WATCHDOG
CONFIG_USB_OTG
CONFIG_MMC_BLOCK

Fixes: 137124260
Fixes: 136278478
Fixes: 136279343
Fixes: 136279418
Fixes: 136279652
Fixes: 136279658
Fixes: 136281307
Change-Id: I630dd2ffd4f6694b497cbb273830d643878694f6
Signed-off-by: Tri Vo <trong@google.com>
---
 arch/arm64/configs/gki_defconfig | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 7c66f40c9739..45644397a2d7 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -274,14 +274,17 @@ CONFIG_HW_RANDOM_VIRTIO=y
 # CONFIG_DEVPORT is not set
 # CONFIG_I2C_COMPAT is not set
 # CONFIG_I2C_HELPER_AUTO is not set
+CONFIG_SPI=y
 CONFIG_SPMI=y
 CONFIG_PINCTRL=y
 CONFIG_PINCTRL_AMD=y
+CONFIG_POWER_AVS=y
 # CONFIG_HWMON is not set
 CONFIG_THERMAL=y
 CONFIG_THERMAL_GOV_USER_SPACE=y
 CONFIG_CPU_THERMAL=y
 CONFIG_DEVFREQ_THERMAL=y
+CONFIG_WATCHDOG=y
 CONFIG_MFD_ACT8945A=y
 CONFIG_MFD_SYSCON=y
 CONFIG_REGULATOR=y
@@ -313,6 +316,7 @@ CONFIG_HID_MICROSOFT=y
 CONFIG_HID_MULTITOUCH=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB=y
+CONFIG_USB_OTG=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_DUMMY_HCD=y
 CONFIG_USB_CONFIGFS=y
@@ -324,16 +328,17 @@ CONFIG_USB_CONFIGFS_F_MIDI=y
 CONFIG_MMC=y
 # CONFIG_PWRSEQ_EMMC is not set
 # CONFIG_PWRSEQ_SIMPLE is not set
-# CONFIG_MMC_BLOCK is not set
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_TRIGGERS=y
+CONFIG_EDAC=y
 CONFIG_RTC_CLASS=y
 # CONFIG_RTC_SYSTOHC is not set
 CONFIG_RTC_DRV_PL030=y
 CONFIG_RTC_DRV_PL031=y
+CONFIG_DMADEVICES=y
 CONFIG_VIRTIO_PCI=y
 # CONFIG_VIRTIO_PCI_LEGACY is not set
 CONFIG_VIRTIO_INPUT=y
@@ -354,6 +359,7 @@ CONFIG_DEVFREQ_GOV_PASSIVE=y
 CONFIG_EXTCON=y
 CONFIG_PWM=y
 CONFIG_GENERIC_PHY=y
+CONFIG_RAS=y
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_INTERCONNECT=y
